MyQueue.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Stack;
+
+class MyQueue {
+
+
+    Stack<Integer> s1;
+    Stack<Integer> s2;
+     
+    /** Initialize your data structure here. */
+    public MyQueue() {
+
+        s1 = new Stack<Integer>();
+        s2 = new Stack<Integer>();
+    }
+    
+    /** Push element x to the back of queue. */
+    public void push(int x) {
+
+        if(s1.isEmpty()){
+            s1.push(x);
+        }
+        else{
+            while(!s1.isEmpty()){
+                s2.push(s1.pop());
+            }
```

Comment:
This is a brute force solution. We discussed O(1) solutions in class, kindly refer to class.

Kindly refer to Sample.java file and enclose Time/Space space complexities along with 3 point explanation of algorithm.

Reply:
Thanks for notifying, I've implemented that version